actions/shared_publish_setup/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: Setups for publishing a package
```

Comment:
This was supposed to be done as a part of https://atlassian.thetradedesk.com/jira/browse/UID2-2681, but I found it's handy to do it as a part of this PR and won't affect the existing shared workflows so I just added here